afs/media/js/views/components/reports/project.js

```diff
@@ -41,6 +41,40 @@ define([
             self.documentationCards = {};
             self.summary = params.summary;
 
+            self.childProjectTableConfig = {
+                ...self.defaultTableConfig,
+                columns: Array(4).fill(null)
+            };
+
+            self.collectionOfChildProjects = ko.observableArray();
+            self.visible = {
+                childProjects: ko.observable(true),
+            };
+
+            const resourceId = ko.unwrap(self.reportMetadata).resourceinstanceid;
+            const relatedResources = async() => {
+                const result = await reportUtils.getRelatedResources(resourceId);
+                const rrs = result?.related_resources;
+                let childProjects =[];
+
+                const relationships = result?.resource_relationships;
+                for (indv_relationship in relationships) {
+                    if (relationships[indv_relationship]?.resourceinstancefrom_graphid ===
+                        relationships[indv_relationship]?.resourceinstanceto_graphid) {
+                            childProjects.push(relationships[indv_relationship]?.resourceinstanceidfrom);
```

Comment:
You need not use a for at all loop here.  Running filter/map on the relationships array defined on line 60 will give you the array you need with less code.